backend/src/main/java/com/bakdata/conquery/apiv1/FilterSearch.java

```diff
@@ -12,16 +18,70 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
-import com.bakdata.conquery.models.concepts.filters.specific.AbstractSelectFilter;
-import com.bakdata.conquery.models.datasets.Dataset;
-import com.bakdata.conquery.models.worker.Namespaces;
-import com.zigurs.karlis.utils.search.QuickSearch;
-
-import lombok.extern.slf4j.Slf4j;
+import static com.zigurs.karlis.utils.search.QuickSearch.MergePolicy.INTERSECTION;
+import static com.zigurs.karlis.utils.search.QuickSearch.UnmatchedPolicy.IGNORE;
 
 @Slf4j
 public class FilterSearch {
 
+	/**
+	 * Enum to specify a scorer function in {@link QuickSearch}. Used for resolvers in {@link AbstractSelectFilter}.
+	 */
+	@AllArgsConstructor
+	@Getter
+	public static enum FilterSearchType {
+		/**
+		 * String must start with search-String.
+		 */
+		PREFIX {
+			@Override
+			double score(String candidate, String keyword) {
+				/* 0...1 depending on the length ratio */
+				double matchScore = (double) candidate.length() / (double) keyword.length();
```

Comment:
So longer results are favored over short results, isn't it better to show the shortest matches first